CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 2.0.0
+
+### Breaking Changes
+
+- [#155](https://github.com/okta/okta-react-native/pull/155) Changes the promise for `signOut` and `signInWithBrowser` methods (iOS)
```

Comment:
How about `Changes `signOut` and `signInWithBrowser` methods signature to return a Promise`